### DISCUSSION TITLE
would it be possible to get update announcements separately for GNU/Linux and mac versions?

### DISCUSSION THREAD

**Discussion by DanielVillarreal** [ImageMagick/ImageMagick#4616] [General]
re: https://download.imagemagick.org/ImageMagick/download/binaries/?C=M;O=D
[ ]	ImageMagick-x86_64-apple-darwin20.1.0.tar.gz	2020-11-23 08:30 	13M	 
[ ]	ImageMagick-x86_64-pc-linux-gnu.tar.gz	2018-09-23 10:54 	12M	 

Would it be possible to get update announcements separately for GNU/Linux and mac versions?
Thank you for all your efforts,
Daniel V.

**Comment by fmw42** [comment]
I believe that they are all (Linux, Mac, Windows) updated at the same time. So one notice should suffice.
